app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -42,7 +42,9 @@
                 tabindex: "-1",
                 "aria-hidden": true,
                 data: data_attributes do %>
-      <% if attachment.document? %>
+      <% if attachment.thumbnail_url %>
+        <%= attachment.thumbnail_url %>
```

Comment:
As per comment below, I'd imagine we want an image element here.

Worth checking whether we have any alt tags normally for this?